### ISSUE TITLE
dump: unable to retrieve field since v3.1.0

### ISSUE THREAD

**Issue by gnuletik** [datocms/js-datocms-client#93]
Hi,

Since `datocms-client` v3.10 upgrade, `dato dump` ignore fields ending with a number.
`v3.2.0` does not fixes the issue.
`v3.0.44` works fine.

We can reproduce the issue with the following configuration file.
On impacted versions, undefined is printed.

dato.config.js
```js
module.exports = (dato, root, i18n) => {
  console.log(dato.content.score1)
  console.log(dato.content.score2)
}
```

**Comment by matjack1** [comment]
thank you @gnuletik for the report. We'll have a look and be in touch as soon as possible

**Comment by souljuse** [comment]
I tried to reproduce but fields ending with numbers do not seem to be the problem. 

@gnuletik I think we should check your specific case! Write to [support](https://www.datocms.com/support) so we can find a solution.

**Comment by gnuletik** [comment]
Fixed by https://github.com/datocms/js-datocms-client/commit/485a627be01c15b94d20bbff9acb5d535c9887db
Available in `v3.2.1`

Thanks for the support @stefanoverna @matjack1 and @souljuse :)
